client/gui/CGuiHandler.h

```diff
@@ -62,6 +64,7 @@ class CGuiHandler
 	               textInterested;
 
 
+	void handleMouseButtonClick(CIntObjectList &interestedObjs, EIntObjMouseBtnType btn, bool isPressed);

```

Comment:
I can later check if clang-format + some quick formatting config would work (separate from this PR).